bezier-rs/docs/interactive-docs/src/utils/drawing.ts

```diff
@@ -13,6 +13,7 @@ export const COLORS = {
 	NON_INTERACTIVE: {
 		STROKE_1: "red",
 		STROKE_2: "orange",
+		STROKE_3: "green",
```

Comment:
So because the standard colors for the X, Y, Z axes are Red, Green, and Blue (RGB -> XYZ) in virtually all software, I'm worried about the maintainability of the Local Extrema example changing colors in the future if we change this color scheme defined here. I'd therefore suggest not using these color consts and hardcoding `"red"` and `"green"` for only the circles and text used in the Local Extrema example. (Then you can remove `STROKE_3` since it isn't used elsewhere yet.)